scripts/installscripts/buster-install-default.sh

```diff
@@ -834,23 +835,46 @@ install_main() {
     echo "${VERSION_NO} - ${COMMIT_NO} - ${USED_BRANCH}" > ${jukebox_dir}/settings/version
     chmod 777 ${jukebox_dir}/settings/version
 
+    sudo mkdir -p /etc/apt/keyrings
+
     # Install required spotify packages
     if [ "${SPOTinstall}" == "YES" ]; then
         echo "Installing dependencies for Spotify support..."
-        # keep major verson 3 of mopidy
-        echo -e "Package: mopidy\nPin: version 3.*\nPin-Priority: 1001" | sudo tee /etc/apt/preferences.d/mopidy
 
         sudo wget -q -O /etc/apt/keyrings/mopidy-archive-keyring.gpg https://apt.mopidy.com/mopidy.gpg
         sudo wget -q -O /etc/apt/sources.list.d/mopidy.list https://apt.mopidy.com/${OS_CODENAME}.list
-        
+
         ${apt_get} update
         ${apt_get} upgrade
-        ${apt_get} install libspotify-dev
-        ${apt_get} ${allow_downgrades} install mopidy mopidy-mpd mopidy-local mopidy-spotify
-        ${apt_get} ${allow_downgrades} install libspotify12 python3-cffi python3-ply python3-pycparser python3-spotify
+        ${apt_get} install libspotify-dev libffi-dev
```

Comment:
Are libspotify-dev and libspotify12 still needed with the new implementation?

Reply:
I don't know where these dependencies came from or which puzzle-piece might need these.